src/assets/stylesheets/layouts/main/index.scss

```diff
@@ -0,0 +1,15 @@
+html,
+body {
+  top: 0;
```

Comment:
I guess you wanted to reset the CSS? If you did, let's use [this](https://meyerweb.com/eric/tools/css/reset/) for standardizing approach.
Unless please skip this comment but don't forget to adjust regarding the above comment